Tests/Indicators/SortinoRatioTests.cs

```diff
@@ -0,0 +1,72 @@
+using NUnit.Framework;
```

Comment:
Would be nice to add some unit tests with a risk rate different than default 0 value too 👍 
`new DateTime()` for it to be more realistic suggest creating a valid datetime on start and just adding a day per `i`
nit
- missing license header
- Should use `var` to the left always when possible

Reply:
These seem unaddressed @JerrodDev